lib/modules/manager/bicep/index.ts

```diff
@@ -4,6 +4,7 @@ export { extractPackageFile } from './extract';
 
 export const defaultConfig = {
   fileMatch: ['\\.bicep$'],
+  filePatterns: ['**/*.bicep'], // not used yet
```

Comment:
Won't match `.bicep` as a filename but not sure if that's a valid name anyway or if it's meant to be like `xyz.bicep`